.travis.yml

```diff
@@ -24,10 +24,12 @@ notifications:
 node_js:
   - "8"
 before_script:
+  - npm run compile
+  - npm run build:package
+  - npm run generate:interacts
   - ./tools/runGanacheCli.sh </dev/null 1>/dev/null 2>&1 &
   - npm run compile:ts
```

Comment:
I would propose to move this one after `npm run compile` ... kind a makes sense to me.

Reply:
I moved `npm run compile:ts` after `npm run generate:interacts`
compile ts will compile index.ts, which imports file from autogenerated interacts, 
So this sequence is needed.